tests/test_tasks.py

```diff
@@ -5,13 +5,14 @@
 
 import pkg_resources
 from parameterized import parameterized
-from utils import UNIT_TEST_CONFIGS, SSLHydraConfig
 from vissl.engines.train import train_main
 from vissl.hooks import default_hook_generator
 from vissl.utils.checkpoint import get_checkpoint_folder
 from vissl.utils.hydra_config import convert_to_attrdict
 from vissl.utils.misc import get_dist_run_id
 
+from utils import UNIT_TEST_CONFIGS, SSLHydraConfig
```

Comment:
one small comment: this file should be untouched in the commit history since we don't need the relative import. Are the changes stemming from the isort ? otherwise let's revert any changes to this file and good to press merge right away then! :)